src/Routes/RosSuggestedInstance/RosSuggestedInstance.js

```diff
@@ -0,0 +1,38 @@
+import React, { useContext, useEffect } from 'react';
+import { PermissionContext } from '../../App';
+import { useChrome } from '@redhat-cloud-services/frontend-components/useChrome';
+import { NotAuthorized } from '@redhat-cloud-services/frontend-components/NotAuthorized';
+import {
+    EmptyStateBody,
+    EmptyState,
+    EmptyStateVariant,
+    Title
+} from '@patternfly/react-core';
+
+export default function RosSuggestedInstance() {
+    const hasPermissions = useContext(PermissionContext);
+    const { hideGlobalFilter, updateDocumentTitle } = useChrome();
+
+    useEffect(() => {
+        updateDocumentTitle('Suggested Instance Types - Resource Optimization | Red Hat Insights');
+        hideGlobalFilter();
+    }, []);
+
+    return (
+        hasPermissions
+            ? <NoEntities />
+            : <NotAuthorized serviceName='Resource Optimization'/>
```

Comment:
```suggestion
            : <NotAuthorized serviceName={ SERVICE_NAME }/>
```

Instead of hardcoading service name everywhere. Could you please declare variable constant?
Example - declare `SERVICE_NAME = 'Resource Optimization'` & use it here